packages/tenant-process/src/services/validators.ts

```diff
@@ -198,9 +189,16 @@ export async function getTenantKindLoadingCertifiedAttributes(
       }
     });
 
-  const attributesIds = getCertifiedAttributesIds(attributes);
-  const attrs = await readModelService.getAttributesById(attributesIds);
-  const extIds = convertAttributes(attrs);
+  const tenantAttributesIds = getCertifiedAttributesIds(attributes);
+  const retrievedAttributes = await readModelService.getAttributesById(
+    tenantAttributesIds
+  );
+  for (const attributeId of tenantAttributesIds) {
+    if (!retrievedAttributes.find((attr) => attr.id)) {
```

Comment:
I don't understand what's happening here, maybe you meant to do this?

```suggestion
    if (!retrievedAttributes.some((attr) => attr.id === attributeId)) {
```

Reply:
I also suggest to use `forEach` instead of `for ... of `

Reply:
Fixed